### PR TITLE
Fix typo (casing issue) inside Getting Started Guide causing a 404 when clicking on the Components link

### PR DESCRIPTION
**Pull request by sync**
## Please verify the following:

- [x] `yarn test` **jest** tests pass with new tests, if relevant
- [x] `README.md` has been updated with your changes, if relevant

## Describe your PR

Addressing this, when reading the Getting Started Guide form Github at https://github.com/infinitered/ignite/blob/master/docs/Guide.md

Screenshot 2022-10-22 at 9 44 27 am

### CODE REVIEW

File: docs/Guide.md [hunk 1]
```diff
@@ -57,7 +57,7 @@ Ignite comes with some prebuilt, flexible, and customizable components. Unlike m
 
 Ignite works fine with other component libraries, but the built-in component system works the best for custom-designed apps.
 
-Check out the [Components](./components.md) documentation.
+Check out the [Components](./Components.md) documentation.
```
Comment (sync) [inline docs/Guide.md:60]: looked at all other references to `components.md` across codebase and they are all pointing at `Components.md`, just this one was not correct.

**Comment by frankcalise** [comment]
Thanks for the catch!

**Review by frankcalise — APPROVED** [review]
LGTM, we just have to take a look at why the CI is acting up. Thanks!

**Comment by joshuayoes** [comment]
@sync can you push up another commit to this branch? 

You can add an empty commit with `git commit --allow-empty -m "Trigger CircleCI job"`

**Comment by sync** [comment]
just did but same result

**Comment by sync** [comment]
feel free to take over this branch, or do the fix directly yourself, it's not big deal if this pr gets closed

**Comment by joshuayoes** [comment]
We don't really need the CI here since no code changes are present. Thanks again for submitting this PR!

**Pull request merged**